tmux-gitmux.tmux

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+BINARY="$CURRENT_DIR/gitmux"
+CONFIG="$CURRENT_DIR/gitmux.conf"
+
+if [ ! -f "$BINARY" ]; then
+  tmux split-window "cd $CURRENT_DIR && go build -o gitmux && echo 'Press any key to continue...' && read -k1"
```

Comment:
This needs to support rebuilding the binary when the plugin is updated.

Reply:
Maybe best would be to not require the user to have the go toolchain, but automatically download the binary release for their arch/os.

Reply:
From what I can tell, tpm just pulls from git when doing an update. I looked through the docs some and don't see a concept of an update hook for us to leverage.

It seems like this may be more complicated than it appears. Look out for scope creep.